### ISSUE TITLE
Allow for labeling notification as SPAM and bookmarking notifications

### ISSUE THREAD

**Issue by AndrewM131** [uchicago-cs/chigame#338]

**Comment by AndrewM131** [comment]
Description: This issue will address users to label a notification as SPAM, and will also allow users to label a notification as a bookmark. By doing this, we can utilize these labels in the future and possibly an inbox for spam and bookmarks.

**Comment by AndrewM131** [comment]
Update: Pull Request has been made, but no review has been made.

**Comment by AndrewM131** [comment]
Closing comment: We did not implement this issue as we wouldn't have enough time to implement the code in time for the deadline of Sprint 3. This will be left for a future team to implement.
The team met once to discuss how to implement this PR, but we were unsuccessful in our attempt to code our implementation, and with it nearing the end of Sprint 3, we decided as a team to leave this to a future team.
